### ISSUE TITLE
Request File Manager

### ISSUE THREAD

**Issue by knkrth** [adonespitogo/AdoBot#21]
Is there is any plan to view the file manger on the adobot-io server of the victim phone?

**Comment by adonespitogo** [comment]
Yes but not my priority as of the moment. I'm open for pull requests.
